applications/GeoMechanicsApplication/tests/test_rotation_with_moving_load.py

```diff
@@ -28,5 +30,33 @@ def test_rotation_with_moving_load(self):
         # updated, the results will be different.
         self.assertAlmostEqual(-0.000177858, value_for_node[2])
 
+    def test_rotation_with_moving_load_constant_system_matrices(self):
+        """
+        This test is the same as test_rotation_with_moving_load, but the system matrices are indicated as constant in
+        the solver settings. Therefore, another builder and solver is used.
+        """
+        test_name = "test_rotation_with_moving_load_constant_system_matrices"
+        file_path = test_helper.get_file_path(test_name)
+
+        # run simulation
+        simulation = test_helper.run_kratos(file_path)
+
+        self.assertTrue(isinstance(simulation._GetSolver().builder_and_solver, KratosGeo.ResidualBasedBlockBuilderAndSolverWithMassAndDamping))
+
+        res_path = os.path.join(file_path, test_name + result_extension)
+
+        reader = test_helper.GiDOutputFileReader()
+        simulation_output = reader.read_output_from(res_path)
+        rotations = simulation_output["results"]["ROTATION"]
+
+        # Validation for Time = 0.02
+        values_time_0_02 = reader.get_values_at_time(1.0, rotations)
+        value_for_node = reader.get_value_at_node(1, values_time_0_02)
```

Comment:
`value_for_node` => `rotations_at_node_1`?

Reply:
Agreed, made the change